src/TestSuite/Fixture/FixtureHelper.php

```diff
@@ -80,7 +80,7 @@ public function loadFixtures(array $fixtureNames): array
                 /** @psalm-var class-string<\Cake\Datasource\FixtureInterface> */
                 $className = $fixtureName;
                 /** @psalm-suppress PossiblyFalseArgument */
-                $name = preg_replace('/Fixture\z/', '', substr(strrchr($fixtureName, '\\'), 1));
+                //unused $name = preg_replace('/Fixture\z/', '', substr(strrchr($fixtureName, '\\'), 1));
```

Comment:
This $name was unused, I wonder if it was meant to be $className assignment instead?

Reply:
Probably a left over from refactoring/cleanup changes. We can remove this.